Dockerfile

```diff
@@ -125,7 +127,7 @@ LABEL description="The authentication client required to expose secrets from a C
 
 # =================== CONTAINER FOR HELM TEST ===================
 
-FROM golang:alpine as k8s-cluster-test
+FROM golang:1.20-alpine as k8s-cluster-test
```

Comment:
Why is this necessary? I don't think we're targeting Go 1.20 in any of the other images

Reply:
Currently `golang:alpine` and `golang:1.20-alpine` are the same image, so there is currently no change to the image.
#502 updated golang in the Dockerfiles to 1.19, a fixed version, and golang:alpine will change over time.
We could use 1.19 however that version is older and that won't help the security issues.
This change is not needed for the version info.